Casks/ngrok.rb

```diff
@@ -1,5 +1,3 @@
-
-// GENERATED BY nd ci agent-homebrew DO NOT EDIT
```

Comment:
Yeah because that's not a comment :P

https://learnxinyminutes.com/docs/ruby/

Reply:
:hot_face: